src/mlpack/tests/cv_test.cpp

```diff
@@ -164,6 +165,33 @@ BOOST_AUTO_TEST_CASE(MSETest)
   BOOST_REQUIRE_CLOSE(MSE::Evaluate(lr, data, responses), expectedMSE, 1e-5);
 }
 
+/**
+ * Test the R squared metric (R2Score).
+ */
+BOOST_AUTO_TEST_CASE(R2ScoreTest)
+{
+  // Making two points that define the linear function f(x) = x - 1
```

Comment:
Could you also add a stop at the end. Thanks.

Reply:
Thanks, @kartikdutt18 wonderful review. One question do we add stop after mathematical statements, as in some case it may cause ambiguity, isn't it? If you still suggest I will change right then.

Reply:
Hey @bisakhmondal, It would be great if you could add them. It's something that I also picked up when I was working on something else.

Reply:
Sure adding. Thanks